cg/cli/upload/scout.py

```diff
@@ -228,6 +253,18 @@ def upload_rna_junctions_to_scout(context: CGConfig, case_id: str, dry_run: bool
     scout_upload_api.upload_rna_junctions_to_scout(dry_run=dry_run, case_id=case_id)
 
 
+@click.command(name="rna-omics-to-scout")
+@DRY_RUN
+@click.argument("case_id")
+@click.pass_obj
+def upload_omics_to_scout(context: CGConfig, case_id: str, dry_run: bool) -> None:
+    """Upload RNA omics files to Scout."""
+    LOG.info("----------------- UPLOAD RNA OMICS TO SCOUT -----------------------")
+
+    scout_upload_api: UploadScoutAPI = context.meta_apis["upload_api"].scout_upload_api
+    scout_upload_api.upload_omics_to_scout(dry_run=dry_run, case_id=case_id)
+
+
```

Comment:
Do you see this being a useful command to run by itself? If not I think the tomte command can just call `scout_upload_api.upload_omics_to_scout` directly

Reply:
rolling back to separate cli commands as passing context is not straightforward without rewriting all other commands common with mip-rna